cnf-certification-test/networking/suite.go

```diff
@@ -121,8 +121,30 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
 		testPartnerSpecificTCPPorts(&env)
 	})
+	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestDpdkCPUPinningExecProbe)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
```

Comment:
change:
testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
to:	
dpdkPods := env.GetCPUPinningPodsWithDpdk()
testhelper.SkipIfEmptyAny(ginkgo.Skip, dpdkPods)
testExecProbDenyAtCPUPinning(&env, dpdkPods)